src/components/Projects.css

```diff
@@ -0,0 +1,367 @@
+.GNOMEproject_div_pos {
+    display: flex;
+    justify-content: space-evenly;
+    flex-wrap: wrap;
+    height: 30rem;
+    width: 100%;
+    padding: 2rem 0rem 2rem 0rem;
+}
+
+.GNOMEDivSize {
+    width: 30rem;
+    height: 25rem;
+}
+
+.GNOMEDivCommonProps {
+    padding: 4px;
+    margin: 8px;
+    background-color: white;
+}
+
```

Comment:
Consider using tailwinds for writing styles, but if it's not possible to create and animate items using tailwinds then only use raw CSS.